estate/models/estate_property.py

```diff
@@ -0,0 +1,50 @@
+import datetime
+
+from odoo import fields, models
+
+
+class EstatePropertyModel(models.Model):
+    _name = "estate.property"
+    _description = "Defines the model of a real estate property"
+
+    name = fields.Char(required=True, string="Title")
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(
+        string="Date of availability",
+        copy=False,
+        default=lambda _: fields.date.today() + datetime.timedelta(days=90),
```

Comment:
```suggestion
        default=fields.Date.add(fields.Date.today(), months=3),
```
I think we can do this. It's cleaner.